brownfield_django/settings_shared.py

```diff
@@ -35,8 +35,8 @@
     'registration',
     'storages',
     'brownfield_django.main',
-    'django_markwhat',
-    'contactus'
+    'contactus',
+    'markdownify',
```

Comment:
I believe this is supposed to be `markdownify.apps.MarkdownifyConfig` according to their doc: https://django-markdownify.readthedocs.io/en/latest/install_and_usage.html

Reply:
Done